docker-compose/kafka.yaml

```
@@ -185,7 +185,7 @@ services:
       # quarkus.log.level: DEBUG
       QUARKUS_OIDC_ROLES_ROLE_CLAIM_PATH: "roles"
       QUARKUS_OIDC_AUTHENTICATION_VERIFY_ACCESS_TOKEN: "true"
-      QUARKUS_OIDC_TOKEN_AUDIENCE: any # should be "schema-registry,kafka" but our keycloak does not set 'aud' on tokens
+      QUARKUS_OIDC_TOKEN_AUDIENCE: schema-registry,kafka
```

According to your insights, should it not be
```suggestion
      QUARKUS_OIDC_TOKEN_AUDIENCE: schema-registry
```